anime/schemas.py

```diff
@@ -2,12 +2,12 @@
 
 AnimeSchema = Schema.from_dict(
     {
-        "anime_id": fields.Int(),
-        "name": fields.Str(),
-        "genre": fields.Str(),
-        "type": fields.Str(),
-        "episodes": fields.Int(),
-        "rating": fields.Float(),
-        "members": fields.Int(),
+        "anime_id": fields.Int(required=True, error_messages={"required": "anime_id is required."}),
```

Comment:
I wonder if those error messages are needed. Looks repetitive. Any other way to do not repeat this part?